README.md

```diff
@@ -125,6 +125,14 @@ You can now launch the build from Xcode and use the iOS simulator or
 your macOS target. At this point the xcframework is not signed.
 
 
+### Debug webviews
+In development builds (run from XCode) it is possible to debug the web-views via Safari development menu.
```

Comment:
Please add a newline here. Github markdown renderrer doesn't care but some renderers do and in plain text it's better